### PR TITLE
[CARBONDATA-1065] Added set command in carbon to update properties dynamically

### PR DESCRIPTION
**Pull request by ravipesala**
The below sql command updates the carbon properties in driver and as well as in executor dynamically in memory.
```
set key1=value1
```

It does not update carbon.properties file, it just updates the running driver and executor properties in memory, so after restart of driver all set properties will be gone.

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Build Success with Spark 1.6.2, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder/2080/

File: integration/spark-common/src/main/scala/org/apache/carbondata/spark/rdd/NewCarbonDataLoadRDD.scala [hunk 1]
```diff
@@ -125,12 +126,16 @@ class SparkPartitionLoader(model: CarbonLoadModel,
...
+  def initialize(addedProperies: util.Map[String, String]): Unit = {
     val carbonPropertiesFilePath = System.getProperty("carbon.properties.filepath", null)
     if (null == carbonPropertiesFilePath) {
       System.setProperty("carbon.properties.filepath",
         System.getProperty("user.dir") + '/' + "conf" + '/' + "carbon.properties")
     }
+    // Add the properties added in driver to executor.
+    CarbonProperties.getInstance().setProperties(addedProperies)
+    // Add the properties added in driver to executor.
+    CarbonProperties.getInstance().setProperties(addedProperies)
```
Comment (xuchuanyin) [inline integration/spark-common/src/main/scala/org/apache/carbondata/spark/rdd/NewCarbonDataLoadRDD.scala]: duplicated...

File: integration/spark/src/main/scala/org/apache/spark/sql/hive/execution/command/CarbonHiveCommands.scala [hunk 2]
```diff
@@ -53,3 +55,15 @@ private[hive] case class DropDatabaseCascadeCommand(dbName: String,
     rows
   }
 }
+
+case class CarbonSetCommand(command: SetCommand)
+  extends RunnableCommand {
+
+  override val output = command.output
+
+  override def run(sparkSession: SQLContext): Seq[Row] = {
```
Comment (xuchuanyin) [inline integration/spark/src/main/scala/org/apache/spark/sql/hive/execution/command/CarbonHiveCommands.scala]: name it `sqlContext` would be better

**Comment by xuchuanyin** [comment]
Hi, I have the following questions:

Will these properties ...

1. be in effect only in current session or until the node restarts ?  `(Effective Scope)`

2. be logged into log file like execution environment variables ?  `(Audit & Tuning)`

**Pull request closed without merging**